src/net.h

```diff
@@ -376,6 +374,8 @@ class CNode
 
     uint64_t nRecvBytes GUARDED_BY(cs_vRecv){0};
 
+    /** Whether this peer provides all services that we want. Used for eviction decisions */
+    std::atomic_bool m_has_all_wanted_services{false};
     std::atomic<std::chrono::seconds> m_last_send{0s};
```

Comment:
nit in b41c44dcdc49941c4c2e70bcdfdcd8b1d956c69f: Maybe move `m_has_all_wanted_services` down to the other eviction stuff (m_last_tx_time, ...). Otherwise, the comment "Used for eviction decisions" will (incorrectly) imply that m_last_send is also used for eviction, when it is currently not.

Reply:
Done.